build/storage/scripts/disk_infrastructure.sh

```diff
@@ -7,7 +7,7 @@
 export DEFAULT_SPDK_PORT=5260
 export DEFAULT_SMA_PORT=8080
 export DEFAULT_NVME_PORT=4420
-export MAX_NUMBER_OF_NAMESPACES_ON_COTROLLER=1024
+export MAX_NUMBER_OF_NAMESPACES_IN_COTROLLER=1024
```

Comment:
watch the spelling - it is CONTROLLER and not COTROLLER

Reply:
My bad, I've changed it.